girder/utility/server.py

```diff
@@ -154,7 +117,6 @@ def configureServer(test=False, plugins=None, curConfig=None):
         'serverRoot': root,
         'serverRootPath': routeTable[constants.GIRDER_ROUTE_ID],
         'apiRoot': root.api.v1,
-        'staticRoot': routeTable[constants.GIRDER_STATIC_ROUTE_ID]
```

Comment:
@girder/developers Should we continue to provide `info['staticRoot']` to plugins? It will always have the value `/static`.